src/main/java/com/ivanmostovyi/demo/repository/MessageRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.ivanmostovyi.demo.repository;
+
+import com.ivanmostovyi.demo.domain.Message;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface MessageRepository extends CrudRepository<Message, Long> {
```

Comment:
This should not be annotated as a `@Repository` as long as real impl bean is registered by the Spring itself.